CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ Change Log
 **Improvements**
 
 - Changed `DJ08` check for `__str__` method to ignore abstract models (@denizdogan)
+- Added `DJ09` - Model must define `class Meta`
+- Added `DJ10` - Class Meta from Model has to define `verbose_name`
+- Added `DJ11` - Class Meta from Model has to define `verbose_name_plural`
+- Fixed some flake8 issues in package and tests and added pypi badge
```

Comment:
Could you update this line to include your name, and also exclude "and added pypi badge" since this has been added on another PR.

```suggestion
- Fixed some flake8 issues in test files (@avallbona)
```

Also please include your username in all the other lines. Thanks

Reply:
done

Reply:
@avallbona the line didn't change at all, maybe you forgot to push the commit?

Reply:
I was still working on the issue, sorry. I've just pushed the commit now.